README.md

```diff
@@ -7,3 +7,12 @@ Managing settings of GitHub repositories.
 Place your GitHub OAuth Token in the file
 `github.token` in the root of this project
 and run the java application.
+
+## New GitHub repositories
+
+If the GitHub Repo Manager need to include 
+a new repository, the following three files 
+need to be updated manually:
+- `github-repo-manager/src/main/java/com/axonivy/github/GitHubRepos.java`
+- `build/raise-deps/raise.sh`
+- `build/raise-version/raise.sh`
```

Comment:
instead of 'code' clocks, you could also do links which leads directly to the correct files even with line numbers ;)